src/clr-core/styles/layout/_type-grid.scss

```diff
@@ -78,55 +78,58 @@
   }
 
   // alignments
-  &[cds-layout~='align-items:top'] {
+  &[cds-layout~='align:top'] {
```

Comment:
Rename `align-items` to `align` the reasoning is `align-items` is a specific CSS property already. Flexbox and Grid have APIs that allow separate alignments to the placement container and the containing items. Because of this we don't want to confuse the API with the underlying existing `align-items`.  To keep the APIs simple will align both the placement container and containing item. Devs can always use CSS to have more control for specific situation to align them separately.